exo-setup/src/docker-setup.ls

```diff
@@ -80,14 +80,23 @@ class DockerSetup
       volumes: rendered-volumes or undefined
 
 
-  _get-external-service-docker-config: ->
+  _get-external-service-docker-config: (done) ~>
     | !@docker-image => throw new Error red "No location or docker-image specified"
-    docker-config = {}
-    docker-config[@role] =
-      image: @docker-image
-      container_name: @role
-      depends_on: @_get-app-dependencies!
-    docker-config
+    DockerHelper.cat-file image: @docker-image, file-name: 'service.yml', (err, external-service-config) ~>
+      | err => throw new Error red "Could not find the configuration for the docker-image"
+      @service-config = yaml.safe-load external-service-config
+      data-path = global-exosphere-directory @app-config.name, @role
```

Comment:
Let's execute line 88-90 only if `@service-config.docker.volumes` exists, like we do in the `_get-service-dependency-docker-config` method.